magento/magento2
# magento/magento2#32990: Date and Time Format for Custom Option Not Working

### Description (*)

Fixed Catalog Date & Time Custom Options not honoring Date Fields Order when JS calendar is used.



### Fixed Issues (if relevant)

1. Fixes https://github.com/magento/magento2/issues/32990

### Manual testing scenarios (*)

1. In admin panel under Stores -> Configuration -> Catalog -> Date & Time Custom Options configure Date Fields Order.
2. Modify any product by adding a Customizable Option of Date type.
3. On frontend verify that observed custom option date of that product matches configuration from step 1.

### Questions or comments


### Contribution checklist (*)
